google/cloud/spanner_v1/database.py

```diff
@@ -223,9 +223,7 @@ def spanner_api(self):
                     channel=grpc.insecure_channel(self._instance.emulator_host)
                 )
                 self._spanner_api = SpannerClient(
-                    client_info=client_info,
-                    client_options=client_options,
-                    transport=transport,
+                    client_info=client_info, transport=transport
```

Comment:
Why we don't pass `client_options` here? Will it cause any problem?

Reply:
Spanner only really uses api_endpoint which we don't need because we have passed it. But I'll add it back in since new options could be supported in future.